pkg/service/dbaas/service.go

```diff
@@ -138,6 +152,15 @@ func (s *Service) fetchManagedDBaaS(ctx *Context) error {
 					"dbaas", managedResource.GetName())
 				continue
 			}
+			if organization, exist := managedResource.GetLabels()[service.OrganizationLabel]; exist {
+				dbaasDetail.Organization = organization
+			} else {
+				// cannot get organization from DBaaS
+				log.Info("Organization label is missing in DBaaS service, skipping...",
+					"label", service.OrganizationLabel,
+					"dbaas", managedResource.GetName())
+				continue
+			}
```

Comment:
Maybe I'm missing something, but wasn't it the whole point to get the organization from the namespace instead of the managed resource?